packages/ui/src/oortLayout/navMenu.tsx

```diff
@@ -35,13 +35,19 @@ export interface INavItems {
         games: NavItemType
         nfts: NavItemType
     }
+    testMinting: {
+        mutation: NavItemType
+        claim: NavItemType
+        faucet: NavItemType
```

Comment:
I think it is better to add faucet to minting and make it optional

Reply:
like this
minting: {
    mutation: NavItemType
    claim: NavItemType
    faucet?: NavItemType
}